src/main/java/dk/kb/util/yaml/YAMLVisitor.java

```diff
@@ -0,0 +1,11 @@
+package dk.kb.util.yaml;
+
+public interface YAMLVisitor {
```

Comment:
Let's talk about [FunctionalInterface](https://docs.oracle.com/javase/8/docs/api/java/util/function/package-summary.html#package.description)s which are made for cases like this one.